tests/foreman/ui/test_errata.py

```diff
@@ -1102,27 +1103,36 @@ def test_positive_content_host_previous_env(
     :Setup:
         1. Make sure multiple environments are present.
         2. Content host's previous environments have additional errata.
+        3. Promote the Host's content view to a new lifecycle environment.
+        4. Set the Host to use the new environment, and original content view.
 
     :steps: Go to Content Hosts -> Select content host -> Errata Tab ->
-        Select Previous environments.
+        Select Previous environments (Environments Dropdown).
 
-    :expectedresults: The errata from previous environments are displayed.
+    :expectedresults: The previous environment name, and content view name are correct.
+        The errata dropdown from previous environments are displayed.
 
     :parametrized: yes
     """
     vm = registered_contenthost
     hostname = vm.hostname
```

Comment:
Instead of defining this variables here, why not use `registered_contenthost.hostname` below?

Reply:
good catch, reusing isn't the best practice here as it could cause confusion for future test edits.